app/controllers/registrations_controller.rb

```diff
@@ -75,13 +88,15 @@ def check_allowed_email(resource)
     resource.errors.add(:email, "is not included in allowed domains.")
   end
 
-  def resolve_profile_field_issues
-    # Run this data update script when in a state of "first user" in the event
-    # that we are in a state where this was not already run.
-    # This is likely only temporarily needed.
-    return unless Settings::General.waiting_on_first_user
-
-    csv = Rails.root.join("lib/data/dev_profile_fields.csv")
-    ProfileFields::ImportFromCsv.call(csv)
+  def build_devise_resource
+    build_resource(sign_up_params)
+    resource.saw_onboarding = false
```

Comment:
Interesting: why are we defaulting this to `true` in the database then set it to `false` here?

Reply:
Maybe the original reasoning was to cover for existing nulls being converted to a safe/sane value when the column was added? There are no null values now (only true and false), and the Users factor already sets this `true` when building users for specs.

Reply:
Yeah I think this, along `editor_version`, can use a default value update.